config.yaml

```diff
@@ -9,6 +9,7 @@ default:
  arch_and_endianness: 'ppc64le'
  release_notes_repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
  release_notes_repo_branch: "master"
+ mock_binary: "/usr/bin/mock"
```

Comment:
Some time ago, we had issues with the mock binary requiring root because we called the command `mock` instead of `/usr/bin/mock`, which lead to a different binary in some machines.
I remember someone commenting that it would be nice to have this configurable, so I think its harmless and maybe useful for someone to have this flexibility.